src/utils/components/floatingPosition.test.jsx

```diff
@@ -12,23 +12,30 @@ class TestComponent extends React.PureComponent {
 			return this.triggerRef;
 		};
 
+		const getContent = () => {
+			return this.contentRef;
+		};
+
 		return(
-			<FloatingPosition
-				getTrigger={getTrigger}
-				noPortal={false}
-				align="right"
-			>
-				{({
-					top,
-					left
-				}) => (
-					<div>
-						<div ref={el => (this.triggerRef = el)}>Trigger</div>
-						<p>top: {top}</p>
-						<p>left: {left}</p>
-					</div>
-				)}
-			</FloatingPosition>
+			<div>
+				<div ref={el => (this.triggerRef = el)}>Trigger</div>
```

Comment:
was this committed by mistake?

Reply:
Nope, I needed to do this so the tests wouldn't error